app/dashboard/configuracion/api/page.tsx

```
@@ -0,0 +1,299 @@
+      setName("")
+      setSelectedChildren([])
+      await loadKeys()
+    } catch (e: any) {
+      toast({ title: "Error", description: e.message, variant: "destructive" })
+    } finally {
+      setCreating(false)
+    }
+  }
+
+  const handleRevoke = async (id: string) => {
+    if (!confirm("¿Revocar esta API key? Dejará de funcionar de inmediato.")) return
```

ESLint

[error] Unexpected confirm.

(no-alert)